fMRI_Tutorial/DataCheck.ipynb

```diff
@@ -0,0 +1,473 @@
+      "text/plain": [
+       "<Figure size 1080x1080 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "df  = pd.DataFrame(matfinal)\n",
+    "matdropped = df.dropna(axis=1, how='all').dropna(axis=0,how='all')\n",
```

Comment:
Great that you keep the same index for the rows/columns, but remove the empty ones. Can't really determine from the code (with my limited python knowledge) whether the matrix itself is actually smaller now, or whether they are merely 'hidden'?

Reply:
it removes these columns/rows from the dataframe, thus final matrix is smaller